index.html

```diff
@@ -146,11 +146,11 @@ <h2>Featured Advisors</h2>
         <h2>Partner</h2>
         <div class="dash">____</div>
         <ul class="partner-div">
-          <li id="barclays">SCIENCE MUSEUM</li>
-          <li id="toyota">ETHIO TELE</li>
-          <li id="star">STAR</li>
-          <li id="oil">OIL</li>
-          <li id="google">GOOGLE</li>
+          <li id="barclays"><img src="/images/mozilla.svg" alt="" /></li>
+          <li id="toyota"><img src="/images/google.svg" alt="" /></li>
+          <li id="star"><img src="/images/naver.svg" alt="" /></li>
+          <li id="oil"><img src="/images/daumkakao.svg" alt="" /></li>
+          <li id="google"><img src="/images/airbnb.svg" alt="" /></li>
         </ul>
```

Comment:
- [ ] Good job by Updating the partner section as required by the previous reviewer but The added images are not visible right now, Please try to fix the import and make sure images are visible inside the webpage. Also, once the images are in place please try to align the contents as they are in the original design. 
### Your Partner section 
![image](https://user-images.githubusercontent.com/69077061/235101115-cf770b6d-ad35-4cb9-ba7e-af4d6884f1be.png)